odata-json-format/15 Delta Payload.md

```diff
@@ -478,17 +492,152 @@ inline as a delta
+      "@Org.OData.Core.V1.DataModificationException": {
+        "failedOperation": "insert",
+        "responseCode": 400,
+        "info": {
+          "code": "incmplt",
+          "message": "Required field(s) not provided",
+          "target": "Address",
+          "com.sap.vocabularies.Common.v1.additionalTargets": [ "Industry", "VATRegistration" ],
+          "severity": "error"
+        }
+      }
+    },
+    {
+      "@Org.OData.Core.V1.ContentID": "2",
+      "CustomerID": "AROUT",
+      "@Org.OData.Core.V1.DataModificationException": {
+        "failedOperation": "update",
+        "responseCode": 400,
+        "info": {
+          "code": "r-o",
+          "message": "Customer is archived and cannot be changed",
+          "severity": "error"
+        }
+      }
+    },
+    {
+      "@Org.OData.Core.V1.ContentID": "3",
+      "CustomerID":"ANTON"
+      "@Org.OData.Core.V1.DataModificationException": {
+        "failedOperation": "delete",
+        "responseCode": 400,
+        "info": {
+          "code": "ufo",
+          "message": "Customer has unfinished orders and cannot be deleted",
+          "severity": "error"
+        }
+      }
+    },
+    {
+      "@Org.OData.Core.V1.ContentID": "4",
+      "CustomerID": "ALFKI",
+      "Orders@delta": [
+        {
+          "@Org.OData.Core.V1.ContentID": "4.3",
+          "@id": "Orders(10835)",
+          "@Org.OData.Core.V1.DataModificationException": {
+            "failedOperation": "update",
+            "responseCode": 400,
+            "info": {
+              "code": "b/s",
+              "message": "RequiredDate cannot be changed because Order is already being shipped",
+              "severity": "error"
+            }
+          }
+        }
+      ]
+    },
+    {
+      "@context": "#Customers/$deletedLink",
+      "@Org.OData.Core.V1.ContentID": "5",
+      "source": "Customers('ANATR')",
+      "relationship":" Orders",
+      "target": "Orders(10643)",
+      "@Org.OData.Core.V1.DataModificationException": {
+        "failedOperation": "link",
+        "responseCode": 404,
+        "info": null
+      }
+    },
+    {
+      "@context": "#Customers/$link",
+      "@Org.OData.Core.V1.ContentID": "6",
+      "source": "Customers('DUMON')",
+      "relationship": "Orders",
+      "target": "Orders(10311)",
+      "@Org.OData.Core.V1.DataModificationException": {
+        "failedOperation": "unlink",
+        "responseCode": 400
+      }
+    }
+  ]
+}
+```
+
+Without the `continue-on-error` preference processing would stop on the first error, and the response would be a standard OData error response
+```json
+HTTP/1.1 400 Bad Request
+Content-Type: application/json
+Content-Length: ###
+
+{
+  "error": {
+    "code": "incmplt",
+    "message": "Required field(s) not provided",
+    "target": "Customers('EASTC')/Address",
+    "com.sap.vocabularies.Common.v1.additionalTargets": [ "Customers('EASTC')/Industry", "Customers('EASTC')/VATRegistration" ]
```

Comment:
Do we want this SAP annotation in the OASIS spec?

Reply:
Yes, explicitly requested in issue https://issues.oasis-open.org/browse/ODATA-1593